app/javascript/onboarding/components/ProfileForm/CheckBox.jsx

```diff
@@ -0,0 +1,35 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { FormField } from '@crayons';
+
+const CheckBox = (props) => {
```

Comment:
refactor (blocking): A general note about components and JS in general. If it's exported from a file, document it via [JSDoc](https://jsdoc.app) comments.

We weren't doing this in the past, but started doing it a little while back. For an example, see https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js

Reply:
@nickytonline I couldn't find good practices on the internet to follow, the examples always deviated slightly. Maybe adding this to our documentation will help developers that are not familiar with jsdoc on how we'd like the functions to be documented?

Reply:
✅